app/lib/shared/handler_helpers.dart

```diff
@@ -171,22 +173,25 @@ shelf.Handler _logRequestWrapper(Logger logger, shelf.Handler handler) {
       if (context.traceId != null) {
         debugHeaders = {'package-site-request-id': context.traceId!};
       }
-      final markdownText = '''# $title
-
-**Fatal package site error.**
-
-$fileAnIssueContent
-
-Add these details to help us fix the issue:
-````
-Requested URL: ${request.requestedUri}
-Request ID: ${context.traceId}
-````
-      ''';
+      final issueUrl = 'https://github.com/dart-lang/pub-dev/issues/new';
+      final message = d.fragment([
+        d.h1(text: title),
+        d.p(child: d.b(text: 'Fatal package site error.')),
+        d.p(children: [
+          d.text('Please open an issue: '),
+          d.a(href: issueUrl, text: issueUrl),
+          d.p(text: 'Add these details to help us fix the issue:'),
+          d.codeSnippet(
+            language: 'text',
+            text: 'Requested URL: ${request.requestedUri}\n'
+                'Request ID: ${context.traceId}',
+          ),
+        ]),
+      ]);
```

Comment:
Are we sure we don't want to prefer to use markdown instead of HTML... it's much harder to read this.

And the risk from rendering a bit of markdown is fairly tiny.

Reply:
We could refactor this into a template method, so that it won't impact the readability of the error processing. I don't see it too bad to have it spelled out though...